server/webserver_test.go

```diff
@@ -0,0 +1,19 @@
+package server
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/livepeer/go-livepeer/core"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSetOrchestratorPriceInfo(t *testing.T) {
```

Comment:
Could we add test cases for an invalid price?

Reply:
fixed in https://github.com/livepeer/go-livepeer/pull/993/commits/54863dce0b699a81e16eb27e09bf64b6b28e79a8